src/containers/Chat/ChatConversations/ChatConversation/ChatConversation.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { shallow } from 'enzyme';
+import ChatConversation from './ChatConversation';
+
+describe('<ChatConversation />', () => {
+  const defaultProps = {
+    contactId: 1,
+    contactName: 'Jane Doe',
+    lastMessage: {
+      body: 'Hello there!',
+      insertedAt: '2020-06-19T18:44:02Z',
+      tags: {
+        id: 1,
+        label: 'Unread',
+      },
+    },
+  };
+  const wrapper = shallow(<ChatConversation {...defaultProps} />);
+
+  test('it should render the name correctly', () => {
+    expect(wrapper.find('[data-testid="name"]').text()).toEqual('Jane Doe');
+  });
+
+  test('it should render the message content correctly', () => {
+    expect(wrapper.find('[data-testid="content"]').text()).toEqual('Hello there!');
+  });
+
+  test('it should render the message date correctly', () => {
+    expect(wrapper.find('[data-testid="date"]').text()).toEqual('00:14');
```

Comment:
is this the right time? considering it's 18:44 in the defaultProps

Reply:
yes, that's right! It is based on the timezone.

Reply:
Hmm... I think we should avoid the timezone as now CI in place, it may have different timezone than IST and the test case may fail. What do you think?

Reply:
I am not checking the actual data, I am checking mock data. So that should be fine. Check the props that are being sent.